InvenTree/company/templates/company/index.html

```diff
@@ -12,12 +12,13 @@
 <h3>{{ title }}</h3>
 <hr>
 
+{% if title == 'Manufacturers' and roles.purchase_order.add or title == 'Suppliers' and roles.purchase_order.add or title == 'Customers' and roles.sales_order.add %}
```

Comment:
This check will fail if the title has been translated from english to another language.

Reply:
Perhaps a different context variable needs to be supplied to the template?

Reply:
Good point, I'll add another variable to the context

Reply:
Actually found `pagetype` very suitable :smiley: